### ISSUE TITLE
[Research] OUI Compliance audit of `src/core/public/chrome`

### ISSUE THREAD

**Issue by BSFishy** [opensearch-project/OpenSearch-Dashboards#4113]
Tracking issue for OUI compliance audit of the `src/core/public/chrome` folder.

Follow the steps in #3945 to complete the audit, then open another issue with findings. Once the work of performing the audit is done, this issue can be closed.

Note that this is a core service, which may be involved with the implementation and inclusion of OUI styles and themes. Since this is a core service, the conclusion of this audit might actually not involve any additional work.

<details>
  <summary>Related files</summary>

size (bytes) | file | notes
---|---|---
2.4K | [src/core/public/chrome/ui/header/header_breadcrumbs.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/core/public/chrome/ui/header/header_breadcrumbs.scss) | 
463 | [src/core/public/chrome/ui/header/_index.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/core/public/chrome/ui/header/_index.scss) | 
384 | [src/core/public/chrome/ui/header/home_loader.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/core/public/chrome/ui/header/home_loader.scss) | 
122 | [src/core/public/chrome/ui/header/collapsible_nav.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/core/public/chrome/ui/header/collapsible_nav.scss) | 
115 | [src/core/public/chrome/ui/header/header_logo.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/core/public/chrome/ui/header/header_logo.scss) | 
86 | [src/core/public/chrome/ui/_loading_indicator.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/core/public/chrome/ui/_loading_indicator.scss) | 
57 | [src/core/public/chrome/ui/_index.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/core/public/chrome/ui/_index.scss) | 
22 | [src/core/public/chrome/_index.scss](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/src/core/public/chrome/_index.scss) | 

</details>

If this issue isn't already assigned to someone and you'd like to take on this work, please ping @BSFishy in the comments and I can assign it to you.
